lib/services/log-parser-service.ts

```diff
@@ -23,22 +24,37 @@ export class LogParserService extends EventEmitter implements ILogParserService
 	@cache()
 	private startParsingLogCore(): void {
 		this.$deviceLogProvider.on(DEVICE_LOG_EVENT_NAME, (message: string, deviceIdentifier: string) => this.processDeviceLogResponse(message, deviceIdentifier));
+		this.$previewSdkService.on(DEVICE_LOG_EVENT_NAME, (message: string, deviceIdentifier: string) => this.processDeviceLogResponse(message, deviceIdentifier));
 	}
 
 	private processDeviceLogResponse(message: string, deviceIdentifier: string) {
-		const device = this.$devicesService.getDeviceByIdentifier(deviceIdentifier);
-		const devicePlatform = device.deviceInfo.platform.toLowerCase();
+		const devicePlatform = this.tryGetPlatform(deviceIdentifier);
```

Comment:
check the performance implications here (we will get the device or catch an exception for each log)